src/main/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtils.java

```diff
@@ -402,22 +403,35 @@ public static String escapeAllControlChars(String text) {
      * Checks Javadoc comment it's in right place.
      * From Javadoc util documentation:
      * "Placement of comments - Documentation comments are recognized only when placed
-     * immediately before class, interface, constructor, method, or field
+     * immediately before class, interface, constructor, method, field or annotation field
      * declarations -- see the class example, method example, and field example.
-     * Documentation comments placed in the body of a method are ignored. Only one
+     * Documentation comments placed in the body of a method are ignored. Only [the last] one
```

Comment:
> Only [the last] one

What is the `[]` suppose to mean here? If you are trying to specify 'the last comment' along with the 'only one', why don't we put it on the end of the sentence.
Ex: `Only one documentation comment per declaration statement is recognized by the Javadoc tool. If there are many comments, it will only recognize the last one.`

Reply:
It is a quote from Javadoc util documentation, as written above. See
https://docs.oracle.com/javase/6/docs/technotes/tools/solaris/javadoc.html#documentationcomments

> Documentation comments are recognized only when placed immediately before class, interface, constructor, method, or field declarations -- see the class example, method example, and field example. Documentation comments placed in the body of a method are ignored. Only one documentation comment per declaration statement is recognized by the Javadoc tool.

The original is not clear. I added an update to the quote. Such additions should be noted to make it clear that this is a modified quote.

Reply:
@rnveach done. The unclear part was extracted from the quote. Now it is a new paragraph.